### PR TITLE
Examples of adding multiple nodes and xref to link added.

### PR DESCRIPTION
**Pull request by jhoyt-rh**
For OED initiative, added link to adding labels to nodes and examples of adding labels to more than one node. Starts at line 858.

### CODE REVIEW

File: install_config/persistent_storage/persistent_storage_ceph_fs.adoc [hunk 1]
```diff
@@ -0,0 +1,393 @@
+[[install-config-persistent-storage-persistent-storage-ceph-rbd]]
```
Comment (vikram-redhat) [inline install_config/persistent_storage/persistent_storage_ceph_fs.adoc]: @jhoyt-rh Hey Joan - is this file supposed to be a part of this PR?

**Comment by vikram-redhat** [comment]
@jhoyt-rh - ok, so could you take this file out? (also please comment directly, not via email :))

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -855,7 +855,18 @@ spec:
...
+For example, if your deployment has three infrastructures, you could add labels for those
+infrastructures as follows:
+----
+oc label node <nodename1> logging-es-node=1
+oc label node <nodename2> logging-es-node=2
+oc label node <nodename3> logging-es-node=3
+----
+
+For information about adding a label to an infrastracture, see
+https://docs.openshift.com/container-platform/3.10/admin_guide/manage_nodes.html#updating-labels-on-nodes.
```
Comment (vikram-redhat) [inline install_config/aggregate_logging.adoc]: @jhoyt-rh should this be an xref to the manage_nodes.adoc file and to the anchor within it?

Comment (jhoyt-rh) [reply install_config/aggregate_logging.adoc]: @vikram-redhat  I'll find out how to xref to the anchor and update it.

**Comment by jhoyt-rh** [comment]
Closed due to git and BZ interaction errors. Going to create a new PR for this BZ.

**Pull request closed without merging**